showcase/karma.conf.js

```diff
@@ -12,16 +12,7 @@ const defaultKarmaConfig = require("./node_modules/@nationalbankbelgium/stark-te
 const karmaTypescriptExclusions = [...defaultKarmaConfig.exclude, "src/assets/examples/**"];
 
 const karmaTypescriptBundlerAliasResolution = {
-	resolve: {
-		alias: {
-			// adapt the resolution of "angular-in-memory-web-api" modules because we don't want to add "@angular/http" to the Showcase npm dependencies!
-			// see https://github.com/angular/in-memory-web-api/issues/215
-			"angular-in-memory-web-api/http-client-in-memory-web-api.module":
-				"./node_modules/angular-in-memory-web-api/bundles/in-memory-web-api.umd.js",
-			"angular-in-memory-web-api/interfaces": "./node_modules/angular-in-memory-web-api/bundles/in-memory-web-api.umd.js",
-			"@angular/http": "./node_modules/@angular/common/bundles/common-http.umd.js"
-		}
-	}
+	resolve: { alias: {} }
```

Comment:
Just a suggestion: can this change be on a separate PR (for example the one about simplifying the karma confs #1145 ) to keep this PR clean with just the changes needed for `ng-packagr`? That way is easier to find relevant changes in the future